corehq/apps/app_manager/tests/test_suite_remote_request.py

```diff
@@ -562,3 +562,62 @@ def test_prompt_default_value(self, *args):
         </partial>
         """
         self.assertXmlPartialEqual(expected, suite, "./remote-request[1]/session/query/prompt[@key='name']")
+
+    def test_cowin_appointment_search(self):
+        self.module = self.app.add_module(Module.new_module("Cowin Vaccination", 'en'))
+        self.app.new_form(0, "Untitled Form", None)
+        self.module.case_type = 'case'
+        suite = self.app.create_suite()
+
+        expected = """
+        <partial>
+          <remote-request>
```

Comment:
<remote-request> must contain a <post> element, you will need to add one with any dummy `url` and set the `relevance` to false

Reply:
hey @shubham1g5 
This should "false()" ?

Reply:
and is the url mandatory?
Can i just have a `post` node with relevant set to false?

Reply:
> This should be "false()" ?

yes

> is the url mandatory?

yes and it needs to be a well formatted PR.  It's fine to put any CC url, since `relevant` is `false()`, CC won't make a request to the url.

Reply:
https://github.com/dimagi/commcare-hq/pull/29867/commits/fe1a1213d1fc5963e7b874dac58aa8a2a89211c7